docs: message for custom fields wildcard (#1052)
salesforcecli/plugin-deploy-retrieve: messages/retrieve.start.md

# wantsToRetrieveCustomFields

> Because you're retrieving one or more CustomFields that you didn't specify the name for, we're also retrieving the CustomObject to which it's associated.

Reviewer: ```suggestion
Because you're retrieving one or more CustomFields, and you didn't specify any field names, we're also retrieving the CustomObject to which they're associated.
```

Reviewer: Okay?

Author: > and you didn't specify any field

nuance: you might have specified **some** field names, but you still have a wildcard on CustomField, too.

Reviewer: So, final message after chatting:

Because you're retrieving one or more CustomFields that you didn't specify the name for, we're also retrieving the CustomObject to which they're associated.
